.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: release
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - '[0-9]*'
+jobs:
+  source:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/cockpit-project/tasks:latest
+      options: --user root
+    permissions:
+      # create GitHub release
+      contents: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Install gh
+        run: dnf install -y gh
+
+      # https://github.blog/2022-04-12-git-security-vulnerability-announced/
+      - name: Pacify git's permission check
+        run: git config --global --add safe.directory /__w/subscription-manager-cockpit/subscription-manager-cockpit
+
+      - name: Get latest tag number
+        run: |
+            echo "TAG_VERSION=$(git describe --tags)" >> $GITHUB_ENV
+
+      - name: Workaround for https://github.com/actions/checkout/pull/697
+        run: git fetch --force origin ${{ env.TAG_VERSION }}:refs/tags/${{ env.TAG_VERSION }}
+
+      - name: Build a release
+        run: make dist node-cache
+
+      - name: Create a release
+        run: |
+          set -x
+
+          git show -s --format=%b ${{ env.TAG_VERSION }} | tail -n +6 > release-note.txt
```

Comment:
magic number, and this is brittle. Moreover, it also includes the PGP signature.

Better command: `git tag -l --format='%(contents:body)' ${{ env.TAG_VERSION }}`

```
❱❱❱ git tag -l --format='%(contents:body)' 250
- Shell: Improve admin switcher and session menu    
- Software Updates: Introduce basic kpatch support
```